masters-thesis.tex

```diff
@@ -216,253 +219,253 @@ \subsection{Lean UX}
+\subsubsection{Market Research Conclusion}
+The team performed both qualitative and quantitative market and competition analysis and was confident that there was a market opportunity to explore, especially considering the demand for solutions outsides of Poland. Evaluating many related existing solutions allowed to explore many ways to implement business model (i.e. pricing and payment options), business domains to explore and user interface ideas and shortcomings. The team started drafting a business model document, and simultaneously to research and implementation efforts, the team looked for a way to legalize their company.
 
-\subsection{First month - Architecture and technologies}
-The team consisted of 6 developers who worked on desktop Java application on daily basis and one developer experienced with PHP and web technologies. The architecture was pretty standard these days - DNCR was decided to be a web app with server, as it would be easiest to distribute and quickly introduce new changes, as well to introduce web search engine later on.
+\subsection{Sprint 0: Architecture and Technologies}
+The team consisted of six developers who worked on the desktop Java application on a daily basis and one developer experienced with PHP and web technologies. The architecture was pretty standard these days: it was decided that the DNCR would be a web app with a server, as it would be easiest to distribute and quickly introduce new changes, as well to introduce a web search engine later on.
 
-For front-end, the team decided to create a Single-Page-App (SPA), as there are many frameworks with great support that would ease learning. Despite Facebook React vast popularity, the team decided on Angular framework from Google (despite it being beta before 2.0 version at that time), as it seemed more opinionated and provided more complete, all-in-one solution that would be easiest to grasp by the team consisting mostly of developers inexperienced with web technologies. Typescript was chosen instead of Javascript, as it was both promoted by Angular and the team used to static typing was enthusiastic about retaining some typing capabilities.
+For the front-end, the team decided to create a Single-Page-App (SPA), as there were many frameworks with great support that would facilitate learning. Despite Facebook React's vast popularity, the team decided on the Angular framework from Google (despite it being beta before 2.0 version at that time), as it seemed more opinionated and provided a more complete, all-in-one solution that would be the easiest to grasp by the team consisting mostly of developers inexperienced with web technologies. The typescript was chosen instead of Javascript, as it was both promoted by Angular and the team used to static typing was enthusiastic about retaining some typing capabilities.
```

Comment:
wróciłem "The typescript" do "Typescript" - to jest język programowania, czyli z dużej i bez "the" - wcześniej jest też samo PHP, Java, w tym samym zdaniu Javascript